Docs/source/running_cpp/parameters.rst

```diff
@@ -277,6 +277,9 @@ Particle initialization
 * ``particles.use_fdtd_nci_corr`` (`0` or `1`) optional (default `0`)
     Whether to activate the FDTD Numerical Cherenkov Instability corrector.
 
+* ``particles.absorbing_bc`` (`0` or `1`) optional (default `0`)
+    Whether to use absorbing boundary conditions for particles, regardless of the boundary conditions for the fields.
```

Comment:
I think the 2nd half sentence might be confusing (or it confuses me). The default will be `0` which means particles get wrapped around or continue propagating outside of the geometry? Is there any leftover dependency of the behavior depending on the field boundary?
```suggestion
    Whether to use absorbing boundary conditions for particles.
```

Reply:
When `particles.absorbing_bc = 0`, particle BC is determined by `geometry.is_periodic = 1`:
- If `geometry.is_periodic = 1` exiting particles will be injected from the other side.
- If `geometry.is_periodic = 0` exiting particles are deleted.

Ideally particle BC would be independent on field BC, but this is out of the scope of this PR.